src/app/interfaces/event.ts

```diff
@@ -24,6 +24,18 @@ export interface CampusEventRequest extends CampusEvent {
 /** RESTful response interface for CampusEvent. */
 export interface CampusEventResponse extends CampusEvent {
   id: number;
+  create_time: string;
```

Comment:
Pls read the code carefully, some of these fields are inherited from its parent interface, there is no need to declare them multiple times.